.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: Report a bug encountered with Scylla Operator
+labels: kind/bug
+body:
+- type: textarea
+  id: problem
+  attributes:
+    label: What happened?
+    description: |
+      Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+  validations:
+    required: true
+
+- type: textarea
+  id: expected
+  attributes:
+    label: What did you expect to happen?
+  validations:
+    required: true
+
+- type: textarea
+  id: repro
+  attributes:
+    label: How can we reproduce it (as minimally and precisely as possible)?
+  validations:
+    required: true
+
+- type: textarea
+  id: operatorVersion
+  attributes:
+    label: Scylla Operator version
+  validations:
+    required: true
+
+- type: textarea
+  id: kubernetesPlatform
+  attributes:
+    label: Kubernetes platform name and version
+    value: |
+      <details>
+      ```console
+      $ kubectl version
+      # paste output here
+      ```
+
+      Kubernetes platform info:
+      </details>
+  validations:
+    required: true
+
+- type: textarea
+  id: must-gather
+  attributes:
+    label: |
+      Please attach must-gather archive. (Must-gather archive is required for every bug report.)
+      See https://operator.docs.scylladb.com/stable/support/must-gather.html to learn how you can collect it.
+      
+      Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
```

Comment:
```suggestion
      Tip: You can attach images or log files by clicking this area to highlight it and then dragging the files in.
```

Reply:
since these aren't specific files, I am not sure `the` belongs there. Also this is taken from GH docs where it is identical https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms